script.py

```diff
@@ -0,0 +1,62 @@
+import pyodbc 
+import pandas as pd
+import smtplib
+import pysftp
+import datetime
+
+
+ftplocation = 'test/in'
+filelocation ="C:/Users/Administrator/Documents/Leads"
```

Comment:
ons moet hierdie generic maak. of n ENVIRONMENT VARIABLE

Reply:
Verseker ENV vars, net nie seker hoe dit op 'n microsoft server werk nie. Sal navorsing doen en uitvind.